src/status_im2/contexts/syncing/setup_syncing/style.cljs

```diff
@@ -46,3 +60,13 @@
    :left              0
    :right             0
    :margin-horizontal 60})
+
+(def title-container-22
+  {:flex-direction      :row
+   :align-items         :center
+   :border-radius       20
+   :padding-horizontal  20
+   :padding-vertical    12
+   :margin-top          12
+   :justify-content     :space-between
+   :background-color    "rgba(42, 74, 245, 0.1)"})
```

Comment:
not sure were you are getting this color but in 99% of use cases the colors should be in `quo2.foundations.colors`